### PR TITLE
Adding three U.Michigan faculties

### PR DESCRIPTION
**Pull request by dfujiki**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] ~~Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.~~

**Adding one or more faculty members (including an entire department)**

- [ ] ~~If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).~~

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] ~~If the institution you are adding is not in the US,
update `country-info.csv`.~~

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -3994,6 +3995,7 @@ Edward P. Tighe,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs/facu
 Edward Sciore,Boston College,http://www.cs.bc.edu/~sciore,NOSCHOLARPAGE
 Edward W. Felten,Princeton University,https://www.cs.princeton.edu/~felten,watx0IAAAAAJ
 Edward W. Knightly,Rice University,http://knightly.rice.edu,-0vALDMAAAAJ
+Edwin Olson,University of Michigan,http://april.eecs.umich.edu/,GwtVjKYAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:3998]: Not the correct home page: https://april.eecs.umich.edu/people/ebolson/

File: csrankings.csv [hunk 2]
```diff
@@ -5751,6 +5753,7 @@ Hongdong Li,Australian National University,http://users.cecs.anu.edu.au/~hongdon
 Hongfei Yan,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=6010,NOSCHOLARPAGE
 Hongfeng Yu,University of Nebraska,http://vis.unl.edu/~yu,yeWsFW4AAAAJ
 Hongjie Chen,Peking University,http://www.sei.pku.edu.cn/people/chenhj,NOSCHOLARPAGE
+Honglak Lee,University of Michigan,http://eecs.umich.edu/~honglak,fmSHtE8AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:5756]: Apparently at Google Brain -- only full-time faculty are eligible for inclusion.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 2 above)
Comment (dfujiki) [reply csrankings.csv:5756]: Thank you for your review!
Honglak Lee is currently an associate professor in UofM. He's teaching a class here.

File: csrankings.csv (hunk 1 above)
Comment (dfujiki) [reply csrankings.csv:3998]: Thanks. Pushed a new commit.

**Comment by fycus-tree** [comment]
Teaching a class isn't enough. See #480 #861 #1319

Or see the list at
https://github.com/emeryberger/CSrankings/blob/gh-pages/old/industry.csv 

Olson and Lee are both primarily appointed elsewhere from my understanding

**Comment by emeryberger** [comment]
Bump @dfujiki

**Comment by dfujiki** [comment]
Thank you for the comments.
Actually, I have been working on this PR with my advisor Reetu Das for writing an internal report to our department. Below is what we discussed. 

To the best of our knowledge:
(1) Honglak Lee met the full-time appointment requirements in Winter'19 - advising Ph.D. students, teaching a class. We are not sure of his summer plans.
(2) Olson is not full-time, so we understand if you don't want to include him. We think he is advising Ph.D. students but does not teach.

We noticed some issues and decided to close the PR.
It does seem tricky how you manage fairness in these situations i.e. can you verify and track every institution and each faculty who is on leave, partial appointment? This information is not available readily on websites. If the faculty is actively advising Ph.D. students, publishing with them, recruiting new Ph.D. students, we think that should be sufficient, but it's the policy.

The other issue is even if we change the time year window for the period when these faculty were full-time, it doesn't show them so it's not showing our active faculties in ML and Robotics.

**Comment by fycus-tree** [comment]
@dfujiki Plenty of the people on the industry list both advise students and have taught a course in 2019 (e.g. Kyunghyun Cho, Ruslan Salakhutdinov)

**Pull request closed without merging**